### PR TITLE
user login and logout. Link task: https://dev.framgia.com/redmine/issues/171950

### PR DESCRIPTION
**Pull request by nguyenphiviet**

### CODE REVIEW

**Comment by nguyenphiviet** [comment]
ready

**Comment by nguyenphiviet** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    end
+  end
+
+  def logged_in?
+    current_user ? true : false
+  end
+
+  def forget user
+    user.forget
+    cookies.delete(:user_id)
```
Comment (chuanhtuan) [inline app/helpers/sessions_helper.rb]: ()

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    cookies.delete(:remember_token)
+  end
+
+  def log_out
+    forget current_user
+    session.delete(:user_id)
```
Comment (chuanhtuan) [inline app/helpers/sessions_helper.rb]: ()

File: app/helpers/sessions_helper.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    @current_user = nil
+  end
+
+  def redirect_back_or default
+    redirect_to(session[:forwarding_url] || default)
+    session.delete(:forwarding_url)
```
Comment (chuanhtuan) [inline app/helpers/sessions_helper.rb]: ()

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -1,20 +1,35 @@
...
-        </li>
-        <li>
-          <%= link_to t(".register"), signup_path %>
-        </li>
-        <li><%= link_to t(".home"), root_path %>
-        </li>
+        <% if logged_in? %>
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+              Account <b class="caret"></b>
```
Comment (chuanhtuan) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/sessions/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      <p class="lead"><%= t ".has_account" %></p>
+      <hr>
+      <%= form_for(:session, url: login_path) do |f| %>
+        <div class="form-group">
+          <%= f.label :email %>
+          <%= f.email_field :email, class: "form-control" %>
+        </div>
+        <div class="form-group">
+          <%= f.label :password %>
+          <!-- #link_to "(forgot password)", new_password_reset_path -->
```
Comment (chuanhtuan) [inline app/views/sessions/new.html.erb]: remove comments

File: app/views/sessions/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+          <%= f.password_field :password, class: "form-control" %>
+        </div>
+        <div class="form-group">
+          <%= f.label :remember_me, class: "checkbox inline" do %>
+            <%= f.check_box :remember_me %>
+            <span>Remember me on this computer</span>
```
Comment (chuanhtuan) [inline app/views/sessions/new.html.erb]: i18n

**Comment by nguyenphiviet** [comment]
ready

**Pull request merged**